content/contributing/developer-guide.md

```diff
@@ -1,3 +1,148 @@
 # Developer Guideline
 
 We're working on it! Stay tuned! Raise PR and that will help us finding good recommendations.
+
+Till then here follows some unsorted tips.
+Neither are a full reference but they give some clue where to look next.
```

Comment:
`These are not a full reference, but they give some clues on where to look next.`